### PR TITLE
79 feature 294 envoi de message

### PR DESCRIPTION
**Pull request by quentinmananes**

### CODE REVIEW

File: src/__tests__/routes/user/chat/newMessage.js
Comment (Exiels) [inline src/__tests__/routes/user/chat/newMessage.js:1]: Rename to use the tests file naming convention

File: src/models/message.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    required: true
+  },
+  content: {
+    type: String,
+    required: true
+  }
+})
+
+// We create answers collection from answersSchema
+const Messages = mongoose.model('messages', messageSchema)
```
Comment (Exiels) [inline src/models/message.js:34]: use camelCase

File: src/models/message.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  date: {
+    type: Date,
+    default: new Date(),
+    required: true
+  },
+  content: {
+    type: String,
+    required: true
+  }
+})
```
Comment (Exiels) [inline src/models/message.js:31]: How do you determine which message is for which chat ?

**Review by Exiels — CHANGES_REQUESTED** [review]
Instead of getting all messages when you get all chats, it is better to get the message of the selected chat.

Otherwise is we have 10 chats but one with like 10000000 messages, the request will be infinite when we want only the chats

File: src/models/message.js (hunk 1 above)
Comment (quentinmananes) [reply src/models/message.js:34]: This line follows the same rules as the others models
image

**Pull request merged**